README.md

```diff
@@ -52,10 +52,60 @@ If you prefer the programming ligature so far, I recommend Fira Code because it
 
 ## Installation
 
+### macOS
+
+1. Download font files
+1. Unzip the archive and install the font:
+1. Drag & Drop the font file to Font Book (or the other font management app)
+
+### Windows
+
 1. Download font files
 1. Unzip the archive and install the font:
-    - macOS: Drag & Drop the font file to Font Book (or the other font management app)
-    - Windows: right-click any of them, then pick `Install` from the menu
+1. right-click any of them, then pick `Install` from the menu
+
+### Debian
+
+The other Linux distributions might be same.
+
+1. clone repository
+
+    ```sh
+    git clone https://github.com/0xType/0xProto.git
+    ```
+
+1. build fonts
+
+    ```sh
+    make
+    ```
+
+1. place files into `${HOME}/.local/share/fonts` directory
+
+    ```sh
+    cp fonts/0xProto-Regular.* ${HOME}/.local/share/fonts/
+    chmod -R 644 ${HOME}/.local/share/fonts/
```

Comment:
Please delete this one line.
The reason is that this directory specification should be optional (e.g., `${HOME}/.font` directory should be fine), and it is not in 0xProto's interest to control access permissions to it.

Reply:
Thank you for the review.

Please allow me to explain about the font search path. 
Currently, the `${HOME}/.fonts` directory should not be used. Because [`${HOME}/.fonts` directory is planed to be removed from the search path in the future by upstream](https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/main/fonts.conf.in#L30-L31). So it needs to use the other directory for user-specific installation. On Debian, it is `${HOME}/.local/share/fonts` directory.

Reply:
> Currently, the ${HOME}/.fonts directory should not be used. Because [${HOME}/.fonts directory is planed to be removed from the search path in the future by upstream](https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/main/fonts.conf.in#L30-L31).

True. I learned something new. Thanks!